metrotranscom/doorway
# fix: remove do you work in

This PR addresses #678 


## Description

This PR removes every reference to workInRegion and subsequently, work address in sites/public, sites/partners, and most of shared-helpers.
Exceptions:

- The backend-swagger.ts file and api folder were not altered.
- testHelpers.ts and applicationData.ts for cypress mock data employ the backend application type which requires the applicantWorkAddress field. 

This approach was discussed with @ludtkemorgan as a way to avoid unnecessary tech debt by only hiding the field from the UI but keeping the backend functionality unchanged. It will likely cause some merge conflicts in the public and partner's folders but given the field doesn't really match the unique approach to jurisdictions that Doorway has shifted to, I believe it is necessary.


## How Can This Be Tested/Reviewed?

This can be tested by trying to break the common app flow, verifying that these questions don't appear on the partners side (paper app add, edit, or view), and general testing around applications.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
